helm_deploy/hmpps-assessments-api/templates/_envs.tpl

```diff
@@ -109,4 +109,19 @@ env:
   - name: APPLICATIONINSIGHTS_CONNECTION_STRING
     value: "InstrumentationKey=$(APPINSIGHTS_INSTRUMENTATIONKEY)"
 
+  - name: REDIS_HOST
+    valueFrom:
+      secretKeyRef:
+        name: {{ .Values.redis.secret }}
+        key: primary_endpoint_address
+
+  - name: REDIS_PASSWORD
+    valueFrom:
+      secretKeyRef:
+        name: {{ .Values.redis.secret }}
+        key: auth_token
+
+  - name: REDIS_TLS_ENABLED
```

Comment:
Looking at the `application.properties` should this environment variable be called `REDIS_SSL`?

Reply:
Pushed fix

Reply:
Awesome, thanks @LucasCairns